README.md

```diff
@@ -1,43 +1,93 @@
-| CPC | A new proposed Top Level Committee – The "Cross Project Council |
-| C3 | A new proposed Top Level Committee – The "Cross Community Council |
-| CNCF | The Cloud Native Computing Foundation (Prior Art) |
-| COC | Code of Conduct |
-| MVF | Minimum Viable Foundation – The bare minimum governance documentation to operate a function joint foundation |
-| Proposal | The staged proposal feedback and iteration process followed in the `/proposals` directory. |
+* architect
+* Intern
+* Mocha
+* Node-Red
+* WebdriverIO
+* webhint
 
-## Deliverables
-The deliverables from this body will be:
+#### At-Large Projects
 
-  - [ ] A recommendations document to help inform the new Foundation's by-laws.
-  - [ ] A recommendations document to help inform the TLCs' charters.
-  - [ ] A collection of fully drafted governance documents required for "Ignition". This is "Minimum Viable Foundation" (MVF).
-  - [ ] A collection of program, initiative, and governance recommendation documents not required for MVF. After Ignition these will be passed to the appropriate bodies (CPC, C3, etc.) to execute.
+* ESLint
+* Esprima
+* Express
+* Grunt
+* HospitalRun
+* Interledger.js
+* JerryScript
+* Libuv
+* Lodash
+* Marko
+* Messageformat
+* Moment
+* PEP
+* QUnit
 
+#### Emeritus Projects
 
-## Making proposals
+* Chassis
+* jQuery Mobile
+* jQuery UI
+* RequireJS
+* Sizzle
+
+## CPC Members
+
+### Voting Members
+
+#### Impact Project Representatives
+
+Each Impact Project may appoint 2 representatives to the CPC as outlined in the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members)
+
+- **Appium**: Dan Graham (@dpgraham) & Jonah Stiennon(@Jonahss)
+- **Dojo**: Matt Gadd (@matt-gadd) & Dylan Schiemann (@dylans)
+- **jQuery**: Dave Methvin (@dmethvin) & Timmy Willison (@timmywil)
+- **Node.js**: Matteo Collina (@mcollina ) & Joe Sepi (@joesepi)
+- **webpack**: Sean Larkin (@TheLarkInn) & Even Stensberg (@evenstensberg)
+
+#### Growth & At Large Project Representatives
+
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the four individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.
+
+- **Architect**: Kris Borchers (@kborchers)
+- **messageformat**: Eemeli Aro (@eemeli)
+- **Mocha**: Christopher Hiller (@boneskull)
+- **Node-RED**: Nick O'Leary (@knolleary)
+- **WebdriverIO**: Christian Bromann (@bromann)
+- **webhint**: Antón Molleda (@molant)
```

Comment:
This lists six people. That can't be right?

Reply:
We ended up with six in the end I believe

Reply:
Ahhhh

https://github.com/openjs-foundation/cross-project-council/issues/152#issuecomment-494943048

I guess that got added in this PR. @christian-bromann it appears that you unfortunately were not added to the CPC voting members during the election period... was there a conversation or another thread that resulted in you being added to the list?

For now I'm going to remove your name but we can add it back if there is a mistake